public/js/test/mochitest/browser_dbg-navigation.js

```diff
@@ -39,4 +38,10 @@ add_task(function* () {
   );
 
   is(countSources(dbg), 4, "4 sources are loaded.");
+
+  // Test that the current select source persists across reloads
+  yield selectSource(dbg, "long.js");
+  yield reload(dbg, "long.js");
+  ok(getSelectedSource(getState()).get("url").includes("long.js"),
```

Comment:
this makes me think you might want to check the scroll location. it's likely we want to persist that as well.

It might be nice to move this to a general check `assertLocation(dbg, "long.js")` in addition to checking the source is selected we should double check that active tab is long.js